static/js/typing_events.js

```diff
@@ -54,10 +58,18 @@ function get_users_typing_for_narrow() {
 export function render_notifications_for_narrow() {
     const user_ids = get_users_typing_for_narrow();
     const users_typing = user_ids.map((user_id) => people.get_by_user_id(user_id));
-    if (users_typing.length === 0) {
+    const num_of_users_typing = users_typing.length;
+
+    if (num_of_users_typing === 0) {
         $("#typing_notifications").hide();
     } else {
-        $("#typing_notifications").html(render_typing_notifications({users: users_typing}));
+        if (num_of_users_typing <= MAX_USERS_TO_DISPLAY_NAME) {
+            $("#typing_notifications").html(render_typing_notifications({users: users_typing}));
+        } else {
+            $("#typing_notifications").html(
+                "<li class='typing_notification'>Several people are typing...</li>",
```

Comment:
This string should be tagged for translation.  

Though I think cleaner would be to just move this conditional into the HTML template rendered by `render_typing_notifications`.

Reply:
I think if you clean up this commit, we can probably just merge this at the beginning of the branch as a small feature / refactoring.

Oh, and use the unicode `...` character; it looks nicer than 3 dots.

Reply:
Do you mean this: https://en.wikipedia.org/wiki/Ellipsis ? It seems to have the dots closer. `__user__ is typing...` too had 3 dots. That too should be changed I think?(perhaps in a different commit)

Since I wasn't sure if you meant the ellipsis character and it looked odd to me, I left the that as three dots.

I cleaned up the commit, moved the conditional to the `.hbs` template, and added it for translation.

Reply:
Replaced the 3-dots with the Unicode … character. It looked odd in the code editor, so I was confused. 

Yes, the … character looks nicer :)
Comparison: (1st is unicode … char, and the second is our previous 3 dots.)
![image](https://user-images.githubusercontent.com/44665669/113758314-529acb00-9731-11eb-95bc-3638f4324f44.png)